database_schemas/cassandra/benchflow.cql

```diff
@@ -87,7 +87,7 @@ CREATE TABLE process (
   to_ignore boolean,
   experiment_id text,
   trial_id text,
-  PRIMARY KEY ((experiment_id, trial_id), source_process_instance_id, process_instance_id)
+  PRIMARY KEY ((experiment_id, trial_id), process_instance_id)
```

Comment:
Why do you delete it again (ref: https://github.com/benchflow/benchflow/pull/29#discussion_r65815412)?